processor/src/main/kotlin/com/livefront/sealedenum/internal/processor/ClassInspectorUtil.kt

```diff
@@ -0,0 +1,26 @@
+package com.livefront.sealedenum.internal.processor
+
+import com.squareup.kotlinpoet.ClassName
+import kotlinx.metadata.isLocal
+
+/**
+ * Converts a [kotlinx.metadata.ClassName] to a [com.squareup.kotlinpoet.ClassName].
+ */
+internal fun createClassName(kotlinMetadataName: kotlinx.metadata.ClassName): ClassName {
```

Comment:
I previously was using `ClassInspectorUtil.createClassName`, which has now been made properly internal, so now I have an equivalent definition here. It might be worth asking this to be exposed upstream, since it seems fairly useful.